spec/requests/registrations_spec.rb

```diff
@@ -75,6 +75,23 @@
 
         expect(response.body).to include("View more sign in options")
       end
+
+      it "creates a user with a random profile image if none was uploaded" do
+        name = "test"
+        # rubocop:disable RSpec/AnyInstance
+        allow_any_instance_of(ProfileImageUploader).to receive(:download!)
+        post "/users", params: {
```

Comment:
Can we please use path helpers instead of hard-coded paths?

```suggestion
        post users_path, params: {
```